mantle/kola/tests/coretest/core.go

```diff
@@ -37,9 +37,18 @@ var offServices = []string{
 	"tcsd.service",
 }
 
+var uefiFirmware = []string{
+	"uefi",
+}
+
+var uefiFirmwareSecure = []string{
+	"uefi-secure",
+}
+
 func init() {
+
 	register.RegisterTest(&register.Test{
-		Name:        "basic",
+		Name:        "basic.bios",
```

Comment:
I think we still need to excluded the firmwares we don't want here.
`ExcludeFirmwares:  uefi and uefi-secure`